api/services/elastigroup/azure/spotVms/paths/vmProtection.yaml

```diff
@@ -6,8 +6,9 @@ post:
   tags:
     - Elastigroup Azure Spot VMs
   parameters:
+    - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
-    - in: path
```

Comment:
wrong change.
the URL is https://api.spotinst.io/azure/compute/group/{groupId}/vm/{**vmName**}/protection